docs/administration.customers.md

```diff
@@ -46,7 +46,7 @@ Partner users can update the status of meshCustomer accounts by clicking the `Ed
 
 #### Disabling Customer Accounts
 
-Operators can disable meshCustomers in case the responsible customer admins do not provide valid [Payment Methods](meshcloud.project-metering.md#payment-methods) or do not maintain correct [metadata](meshstack.tag-schema.md).
+Operators can disable meshCustomers in case the responsible customer admins do not provide valid [Payment Methods](meshcloud.payment-methods.md) or do not maintain correct [metadata](meshstack.tag-schema.md).
```

Comment:
We should mention how it is possible to disable a customer

Reply:
I wouldn't know how :P I just updated the link here

Reply:
checked with stefan but it is really hacky ;-) you can set the customer to "not verified" but this is maybe not working anymore. There is also documentation regarding this functionality here [https://docs.dev.meshcloud.io/docs/administration.customers.html#disabling-customer-accounts](url) I think we should think about having that as a roadmap item. Will get back to you